src/views/RichWorkspace.vue

```diff
@@ -149,7 +149,7 @@ export default {
 				return
 			}
 			this.creating = true
-			window.FileList.createFile('Readme.md', { scrollTo: false, animate: false }).then((status, data) => {
+			window.FileList.createFile(t('text', 'Readme') + '.md', { scrollTo: false, animate: false }).then((status, data) => {
```

Comment:
Mind to explain that change :thinking: You want to support 'Liesmich.md` as well :rofl:

Reply:
This is mainly fixing the behavior I assumed I had already implemented :see_no_evil: 

See https://github.com/nextcloud/text/issues/727#issuecomment-608028652 for the discussion on the translated file name :wink: 

I also find the german "Anleitung.md" a bit odd, but also agree with @jancborchardt that Readme.md is a quite technical description that might confuse users.

Reply:
Oh :) I think translating the file name is a mistake. You add a lot of complexity and weirdness but fine by me.

Reply:
A note: I think using a translated `Readme` to create a file is bad.

The current assumption is that everyone uses the same language:

1. We have Alice (lang: german) and Bob (lang: english)
2. Alice creates a folder and uses rich workspace. A file `Anleitung.md` is created.
3. Alice shares the Folder to Bob.
4. Bob does not see rich workspace because `Anleitung.md` is not a valid file name for him. 
5. Bob also uses rich workspace. A file `Readme.md` is created. 
6. They both edit rich workspace but do not see their changes ;)

Reply:
That is a very good point actually. Then we probably should keep Readme.md as the default and just allow the translated name to be picked up if such a file exists. What do you think @jancborchardt 

I removed this commit from the PR now since it is actually unrelated to the fix for #807 it just was my first trace to the bug. ;)

Reply:
> What do you think @jancborchardt

I always think that the task of software is to reduce complexity towards people using the software – and yes that means more complexity in the code, which is fine. :) It will be very strange to any non-English speaker, especially non-native, if a file called "Readme.md" shows up in their file list.


------

> Every application has an inherent amount of irreducible complexity. The only question is: Who will have to deal with it—the user, the application developer, or the platform developer?

See https://en.wikipedia.org/wiki/Law_of_conservation_of_complexity

Reply:
And just to clarify: I do realize this would probably be a ton of work, so feel free to put it very far down on the todo list or not on it at all. ;)
(Also, there might be confusion the other way around when someone on the team says "Check the readme" and it’s called differently for everyone.)